python/pyspark/sql/connect/plan.py

```diff
@@ -466,6 +466,7 @@ def __init__(
     def plan(self, session: "SparkConnectClient") -> proto.Relation:
         assert self._child is not None
         plan = proto.Relation()
+        plan.deduplicate.input.CopyFrom(self._child.plan(session))
```

Comment:
this is a case that probably we have a test in `test_connect_basic` to avoid, maybe BTW add a test case there?

Reply:
The existing tests are exhaustive they just missed that the input was never copied.

Reply:
I won't block this PR by my comment.

Reply:
Let me just merge this for now and go forward.